binding/unity/PorcupineManager.cs

```diff
@@ -0,0 +1,155 @@
+﻿//
+// Copyright 2021 Picovoice Inc.
+//
+// You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+// file accompanying this source.
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+//
+
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+using UnityEngine;
+using UnityEngine.UI;
+
+
+namespace Pv.Unity {
+
+    public class PorcupineManager
```

Comment:
Should this class have its own finalizer?

Reply:
I suppose if it becomes unreachable, then so does its Porcupine resources, which then run the finalizers. So perhaps not...